apps/condo/domains/acquiring/schema/RegisterMultiPaymentForVirtualReceiptService.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * Generated by `createservice acquiring.RegisterMultiPaymentForVirtualReceiptService`
+ */
+const { get, isNil } = require('lodash')
+const Big = require('big.js')
+
+const { getById, GQLCustomSchema } = require('@condo/keystone/schema')
+
+const access = require('@condo/domains/acquiring/access/RegisterMultiPaymentForVirtualReceiptService')
+const { DEFAULT_MULTIPAYMENT_SERVICE_CATEGORY } = require('@condo/domains/acquiring/constants/payment')
+const {
+    FEE_CALCULATION_PATH,
+    WEB_VIEW_PATH,
+    DIRECT_PAYMENT_PATH,
+} = require('@condo/domains/acquiring/constants/links')
+const { Payment, MultiPayment, AcquiringIntegration } = require('@condo/domains/acquiring/utils/serverSchema')
+const {
+    getAcquiringIntegrationContextFormula,
+    FeeDistribution,
+} = require('@condo/domains/acquiring/utils/serverSchema/feeDistribution')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@condo/keystone/errors')
+const { DV_VERSION_MISMATCH, WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
+const { checkDvSender } = require('@condo/domains/common/utils/serverSchema/validators')
+
+const {
+    RECEIPTS_ARE_DELETED,
+    RECEIPTS_HAVE_NEGATIVE_TO_PAY_VALUE,
+    ACQUIRING_INTEGRATION_DOES_NOT_SUPPORTS_BILLING_INTEGRATION,
+    RECEIPT_HAS_DELETED_BILLING_INTEGRATION,
+    BILLING_INTEGRATION_ORGANIZATION_CONTEXT_IS_DELETED,
+    ACQUIRING_INTEGRATION_IS_DELETED,
+    CANNOT_FIND_ALL_BILLING_RECEIPTS,
+    ACQUIRING_INTEGRATION_CONTEXT_IS_DELETED,
+} = require('../constants/errors')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const errors = {
+    DV_VERSION_MISMATCH: {
+        mutation: 'registerMultiPaymentForVirtualReceipt',
+        variable: ['data', 'dv'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Wrong value for data version number',
```

Comment:
You can also use `messageForUser` field to provide translated error message text.
Take a look at `RegisterNewUserService.js`.

Reply:
This is not a user faced service. It designed to be called through a condo middleware (a code inside route handler)